tutorials/06-physics-simulation-concepts.md

```diff
@@ -0,0 +1,241 @@
+
+```bash
+ign gazebo ~/diff_drive.sdf
+```
+
+To see the pose changing when the simulation running, we publish a
+\ref ignition::msgs::Twist "Twist" message to command the `vehicle_blue` car to
+move in a circle with 2 meters radius as follow:
+
+```bash
+ign topic -t "/model/vehicle_blue/cmd_vel" -m ignition.msgs.Twist -p "linear: {x: 1.0}, angular: {z: 0.5}"
+```
+
+Then please press the Play button to start the simulation.
+This command tells the car to move in its coordinate frame with velocity
+1.0 meter per second in the X-axis and angular velocity of 0.5 radians per second in
+Z-axis.
+
+Note that the mechanism to move the car is different depending on the used physics
+engine. Using [dartsim](https://github.com/ignitionrobotics/ign-physics/tree/master/dartsim),
+the car is moved by applying force on the joints, please see [DiffDrive.cc](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/src/systems/diff_drive/DiffDrive.cc#L333) code.
+Using [TPE](https://github.com/ignitionrobotics/ign-physics/tree/main/tpe),
+TPE directly sets model velocity in [VelocityControl.cc](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/src/systems/velocity_control/VelocityControl.cc#L117) code.
+
+#### Monitoring the model and its links pose
+
+We can monitor the model pose by selecting the moving car and then
+select the drop-down list `Pose`. Moreover, we could also read the model
+links' poses relative to their parent link by selecting the
+corresponding link on the model tree:
+
+<img src="https://user-images.githubusercontent.com/18066876/97189395-d45ef380-17a4-11eb-9c29-cf33b5b184c4.gif"/>
+
+Note that using the model tree as shown in the above gif, we can view the
+parameters and properties such as `visual` or `collision` of each link or joint
+in the model. For more detail about the kinematics of the model tree, see this
+[tutorial](http://sdformat.org/tutorials?tut=spec_model_kinematics&cat=specification&).
+
+#### Visualizing collision
+
+One of the most wonderful features of physics simulation is the capability to
+simulate collision. We can do a fun experiment like this: while the `vehicle_blue`
+car is moving in a circle, we move the `vehicle_green` to be on `vehicle_blue`'s
+upcoming path. We will see the blue car will push the green car!
+
+<img src="https://user-images.githubusercontent.com/18066876/97190565-28b6a300-17a6-11eb-9ed5-0d6d50c15ad4.gif"/>
+
+To see where these collision parameters are set in SDFormat and how it works,
+please see this [tutorial](http://sdformat.org/tutorials?tut=spec_shapes&cat=specification&).
+
+### Lift Drag
+
+This model shows how joint force, torque, and pressure are supported in Ignition Physics.
+Please download the Lift Drag model to your home folder by:
+
+```bash
+wget https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/main/examples/worlds/lift_drag.sdf -P ~
+```
+
+Like above, please start the Lift Drag model on Ignition Gazebo by:
```

Comment:
Nit: a model refers to a single car or robot in a world, in this case we are using a entire simulation world with light, grid, plane and all those prebuilt into the sdf file. 
```suggestion
Like above, please start the Lift Drag demo world on Ignition Gazebo by:
```